### ISSUE TITLE
[Label]: missing accessibility properties for "required" attribute

### ISSUE THREAD

**Issue by anshul-031** [SAP/ui5-webcomponents#6863]
### Describe the bug

Screen readers don't read "required" for the "Label" web component that are enabled required

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-label--docs

### Reproduction steps

1. Set property 'required' of Label as true.
2. Asterik(*) will come for the Label
3. The screen render don't read out the 'required' word

### Expected Behaviour

It should read out 'required' if some field is set as required

### UI5 Web Components for React Version

1.8.0

### UI5 Web Components Version

1.10.3

### Browser

Chrome, Edge

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by yanaminkova** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

Could you please confirm whether "required" should be part of the speech output when the `Label` is marked as `required`

Thanks and Kind Regards,
Yana

**Comment by anshul-031** [comment]
If this "required" is not needed to be mandatory part of "speech output". Is there a attribute possible for handling this or enable this?
For example something like "requiredAccessible=true" or "requiredAccessible=false" through which we can customize it.

Regards,
Anshul

**Comment by LidiyaGeorgieva** [comment]
Hello @anshul-031 ,

The proper use of ui5-label is to define a label for elements like input, textarea, select. In that case the ui5-label is read when the focus goes to the bound element. For that the "*" (required) should not be read, as it is just a decorative element. It should not be read even with VPC cursor mode.

ui5-label is not supposed to be used alone. Maybe samples and documentation should be improved to show that.

Regards,
Lidiya
